app/src/main/java/com/zaffierce/taskplanner/AddTaskActivity.java

```diff
@@ -0,0 +1,14 @@
+package com.zaffierce.taskplanner;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+public class AddTaskActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_addtask);

```

Comment:
Lab requires a success message when user submits a new task.  So, make an onClickListener for the "Add Task" button like you did in your MainActivity.  You could use a Toast message like this if you want: `Toast.makeText(getApplicationContext(), "Task added" ,Toast.LENGTH_LONG).show();`